File: api/DataStore/ChatacterVoiceSetting/TtsSoftWareVoiceSettingReq.py


from api.gptAI.HumanInformation import CharacterId,CharacterSaveId
from pydantic import BaseModel


class TtsSoftWareVoiceSettingReq(BaseModel):
    page_mode: str
    client_id: str
    character_id: CharacterId
    saveID: CharacterSaveId

File: app-ts/src/ZodObject/DataStore/ChatacterVoiceSetting/TtsSoftWareVoiceSettingReq.ts
import { z } from "zod";
export const TtsSoftWareVoiceSettingReq = z.object({
  page_mode: z.string(),
  client_id: z.string(),
  character_id: z.string(),
  saveID: z.string(),
});
export type TtsSoftWareVoiceSettingReq = z.infer<
  typeof TtsSoftWareVoiceSettingReq
>;
